plugins/utils/neocord.nix

```diff
@@ -0,0 +1,201 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+  helpers.neovim-plugin.mkNeovimPlugin config {
+    name = "neocord";
+    defaultPackage = pkgs.vimPlugins.neocord;
+
+    maintainers = [maintainers.redyf];
+
+    imports =
+      map
+      (
+        optionName:
+          mkRemovedOptionModule
+          ["plugins" "neocord" optionName]
+          "Please use `plugins.neocord.settings` instead."
+      )
+      [
+        "autoUpdate"
+        "logo"
+        "logoTooltip"
+        "mainImage"
+        "clientId"
+        "logLevel"
+        "debounceTimeout"
+        "enableLineNumber"
+        "blacklist"
+        "buttons"
+        "fileAssets"
+        "showTime"
+        "globalTimer"
+        "editingTimer"
+        "fileExplorerText"
+        "gitCommitText"
+        "pluginManagerText"
+        "readingText"
+        "workspaceText"
+        "lineNumberText"
+        "terminalText"
+      ];
+
```

Comment:
You can remove all that. This creates deprecation errors for those options.
But, as this module has never existed in nixvim, there is no need for warning the user about deprecated options.